src/praisonai/praisonai/integrations/db_session_adapter.py

```
@@ -41,11 +41,66 @@ def __init__(self, db: Any):
+        try:
+            from ..persistence.conversation.base import ConversationSession
+
+            session = conv.get_session(session_id)
+            if session is None:
+                conv.create_session(
+                    ConversationSession(session_id=session_id, metadata=dict(meta))
+                )
+            else:
+                session.metadata = {**(session.metadata or {}), **meta}
+                conv.update_session(session)
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)